R/dependencies.R

```diff
@@ -0,0 +1,112 @@
+##' @title Get the dependencies for a given report from the orderly_run.yml
+##' 
+##' @param report the name of the report
+##' @param id the id of the report, if omitted, use the id of the latest report
+##' @param config_path Path to the orderly configuration
+##' @param draft If true look in the draft directory otehrwise look in archive
+##'
+##' @export
+get_dependencies <- function(name, id = NULL, config_path = NULL, draft = FALSE) {
+  if (is.null(id))
+    id <- orderly:::orderly_latest(name, config = config_path, draft = draft)
+  
+  if (is.null(config_path))
+    config_path <- "."
+
+  if (draft)
+    rds_path <- file.path(config_path, "draft", name, id, "orderly_run.rds")
+  else
+    rds_path <- file.path(config_path, "archive", name, id, "orderly_run.rds")
+
+  if (!file.exists(rds_path)) {
+    warning(sprintf(
```

Comment:
I generally dislike warnings. This should probably be an error (there is one other `warning` in all of orderly)